.github/workflows/windows-amd64.yml

```diff
@@ -0,0 +1,655 @@
+          install: >-
+            git
+            make
+            patch
+            pkg-config
+            tar
+            unzip
+            zip
+      - uses: actions/download-artifact@v2
+        with:
+          name: ${{ needs.tensorflow_opt-Windows.outputs.cache_key }}
+          path: ${{ github.workspace }}/
+        if: needs.tensorflow_opt-Windows.outputs.status == 'missing'
+      - uses: ./.github/actions/check_artifact_exists
+        with:
+          name: ${{ needs.tensorflow_opt-Windows.outputs.cache_key }}
+          path: ${{ github.workspace }}/
+          download: true
+        if: needs.tensorflow_opt-Windows.outputs.status == 'found'
+      - run: |
+          "C:/Program Files/7-Zip/7z.exe" x home.tar.xz -so | "C:/Program Files/7-Zip/7z.exe" x -aos -si -ttar -o`pwd`
+          rm home.tar.xz
+      - run: |
+          git status
+      - run: ./ci_scripts/host-build.sh ${{ matrix.build-flavor }}
+      - run: ./ci_scripts/package.sh
+      - uses: actions/upload-artifact@v2
+        with:
+          name: "native_client.${{ matrix.build-flavor }}.tar.xz"
+          path: ${{ github.workspace }}/artifacts/native_client.tar.xz
+      - uses: actions/upload-artifact@v2
+        with:
+          name: "libdeepspeech.${{ matrix.build-flavor }}.zip"
+          path: ${{ github.workspace }}/artifacts/libdeepspeech.zip
+  build-python-Windows:
+    name: "Build Python bindings for Windows"
+    runs-on: windows-2019
+    needs: [build-lib_Windows, swig_Windows]
+    strategy:
+      matrix:
+        build-flavor: ["tf", "tflite"]
+        python-version: [3.6.8, 3.7.9, 3.8.8, 3.9.2]
+    steps:
+      - name: Switch git-bash shell to MSYS2 shell by adding MSYS2 path to PATH front
+        run: echo "D:\a\_temp\msys\msys64\usr\bin" >> $GITHUB_PATH
+      - uses: mozilla/setup-msys2@v2
+        with:
+          msystem: MSYS
+          path-type: inherit
+          update: true
+          install: >-
+            make
+      - uses: mozilla/msvc-dev-cmd@v1
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: actions/download-artifact@v2
+        with:
+          name: "native_client.${{ matrix.build-flavor }}.tar.xz"
+          path: ${{ github.workspace }}/tensorflow/bazel-bin/native_client/
+      - run: |
+          pushd tensorflow/bazel-bin/native_client/
+          "C:/Program Files/7-Zip/7z.exe" x native_client.tar.xz -so | "C:/Program Files/7-Zip/7z.exe" x -aoa -si -ttar -o`pwd`
+          ls -hal
+          popd
+      - uses: actions/download-artifact@v2
+        with:
+          name: "swig_Windows"
+          path: ${{ github.workspace }}/native_client/ds-swig/
+      - name: Link ds-swig into swig
+        run: |
+          set -ex
+          ls -hal native_client/ds-swig/bin
+          ln -s ds-swig.exe native_client/ds-swig/bin/swig.exe
+          chmod +x native_client/ds-swig/bin/ds-swig.exe native_client/ds-swig/bin/swig.exe
+      - uses: actions/setup-python@v2
```

Comment:
I switched to a hand-crafted action to make sure the ABI was older than macOS 10.14, what is the smallest ABI it is compatible with here?

Reply:
No idea.